auth/src/main/res/layout/register_email_layout.xml

```diff
@@ -66,7 +66,7 @@
             style="@style/FirebaseUI.Text.BodyText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:layout_marginTop="336dp"
+            android:layout_marginTop="@dimen/register_field_padding_vert_3x"
```

Comment:
The comment above in the source says:

```
 Since the TextInputLayouts change size based on whether or not an error
 is showing we use a relative layout to position them absolutely
```

So for that reason I am hesitant to change it.  Clearly Aaron found something better about the LinearLayout when trying to match the mocks.

I may be able to replace with a FrameLayout though since we're not using any of the RelativeLayout's relational bits?  I'll try.